### ISSUE TITLE
viewing the generated html files?

### ISSUE THREAD

**Issue by sbc100** [WebAssembly/wasi-proposal-template#6]
Perhaps a dumb question, but is there an easy to view the auto-generated html files?

Should we publish them to github pages perhaps?

If they are not going to viewable, is it even worth checking them into the repository at all?

**Comment by sunfishcode** [comment]
No, there's not currently a way to view the generated html files directly from the github UI. I agree, we should either find a way to make this work or remove the files.

**Comment by sunfishcode** [comment]
I've submitted a fix for this in #7. The generated HTML could stand to be prettier, but at least that PR makes it viewable :-).
